specifications.yml

```diff
@@ -877,6 +901,34 @@ components:
           description: "List of reminders and fees in case of late payment of the invoice"
           items:
             $ref: "#/components/schemas/dunning"
+        creditInvoices:
+          type: array
+          description: "List of credit invoices"
+          items:
+            $ref: "#/components/schemas/creditInvoice"
+
+    creditInvoice: 
+      type: object
+      properties:
+        amount:
+          type: number
+          description: Amount of the credit invoice
+        date:
+          type: string
+          format: date
+          description: Registration date of the credit invoice
+        referenceNumber: 
+          type: string
+          description: Payment reference number of invoice to be used when paying
```

Comment:
Probably not "to be used when paying" since you do not pay credit invoices :smile:

Reply:
Fixed :+1: